examples/MuscularSnake/muscular_snake.py

```diff
@@ -0,0 +1,400 @@
+__doc__ = """Muscular snake example from Zhang et. al. Nature Comm 2019 paper."""
+import sys
+
+sys.path.append("../../")
```

Comment:
remove?

Reply:
### 
For now lets keep it after creating separate example repo, we can remove this. I think otherwise there will be path problem